client/v2/autocli/testdata/help-echo.golden

```diff
@@ -27,6 +27,9 @@ Flags:
   -h, --help                                                                 help for echo
       --i32 int32                                                            some random int32
       --i64 int                                                              
+      --map-string-coin map[string]cosmos.base.v1beta1.Coin                  some map of string to coin
+      --map-string-string stringToString                                     some map of string to string (default [])
+      --map-string-uint32 stringToUint32                                     some map of string to int32
```

Comment:
could we drop the string, i think --map-coin,string,uint32 is enough

Reply:
Sure make sense

Reply:
This still needs to be updated.